bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/internal/AttributesJson.kt

```diff
@@ -28,22 +29,22 @@ private fun toAttributeJson(value: Any, json: JsonWriter) {
             is Long, is Int, is Short, is Byte -> {
                 json.name("intValue").value(value.toString())
             }
-            is Collection<*> -> writeArray(json, value.iterator())
-            is Array<*> -> writeArray(json, value.iterator())
+
+            is Collection<*> -> writeArray(json, value.iterator(), spanName, key)
+            is Array<*> -> writeArray(json, value.iterator(), spanName, key)
             is IntArray -> writeIntArray(json, value)
             is LongArray -> writeLongArray(json, value)
             is DoubleArray -> writeDoubleArray(json, value)
+            else -> Logger.w("Unexpected value in attribute $key attribute of span $spanName")
```

Comment:
This should handle cases where the `spanName` is `null` since then the attributes are not coming from a `Span` (ie: resource attributes or similar).

I think we should also quote the `key` and `spanName` to avoid any confusion when an attribute key, or span name has spaces or otherwise looks like a normal part of the log message. ie `"Unexpected value in attribute '$key'...`)